src/main/java/seedu/address/model/person/BloodType.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represent's a Person's blood type in the address book.
+ *
```

Comment:
The class header comment is not consistent with the class header comment like `Phone` or `Email`